frontend/apps/reader/modules/readersearch.lua

```diff
@@ -21,14 +21,20 @@ end
 function ReaderSearch:addToMainMenu(menu_items)
     menu_items.fulltext_search = {
         text = _("Fulltext search"),
-        tap_input = {
-            title = _("Input text to search for"),
-            ok_text = _("Search all text"),
-            type = "text",
-            callback = function(input)
-                self:onShowSearchDialog(input)
-            end,
-        },
+        callback = function()
+            self:onShowSearchInput()
```

Comment:
`readersearch.lua:25: attempt to call method 'onShowSearchInput' (a nil value)`

Reply:
My bad, I changed the name last second because I thought it sounded too generic this way.